src/configure

```diff
@@ -42,7 +42,7 @@
 
 # This should be incremented after any significant change to the configure
 # script, i.e. any change affecting kaldi.mk or the build system as a whole.
-CONFIGURE_VERSION=7
+CONFIGURE_VERSION=8
```

Comment:
@galv, the configure version is already 8 now (and I don't know why the diff isn't vs. the master).  Anyway-- I think if you just bump it up to 9, we should be good to go with this.  Let's not overthink it.